apps/bfd-server/bfd-server-war/src/test/resources/endpoint-responses/v2/claimResponseSearchPaginated.json

```diff
@@ -0,0 +1,118 @@
+{
+  "resourceType": "Bundle",
+  "id": "1856fb88-119e-46b6-a775-ab78ae0f61e9",
+  "meta": {
+    "lastUpdated": "1970-01-01T00:00:01.000+00:00"
+  },
+  "type": "searchset",
+  "link": [
+    {
+      "relation": "first",
+      "url": "https://localhost:6500/v2/fhir/ClaimResponse?mbi=a7f8e93f09&service-date=gt1970-07-18&service-date=lt1970-07-30"
```

Comment:
@KyRobbins why are the `_count` and `startIndex` params missing from these links?

Reply:
I copied this data from a different query so the parameters aren't quite right, but being that the url is completely ignored during the test diff (because the port is random), it didn't seem it would cause any issues.